### ISSUE TITLE
[FEATURE] Support JSON logging (logstash)

### ISSUE THREAD

**Issue by MarianZoll** [SparebankenVest/azure-key-vault-to-kubernetes#88]
**Is your feature request related to a problem? Please describe.**
When it comes to next gen operations with Azure Monitor and ALA it would be nice to have the logs being written to stdout in a JSON format. This would also ease operations of akv2k8s for ELK and all other ops stacks which use the standard way of doing things.

**Describe the solution you'd like**
Have an option available as a configuration parameter settable as a Helm value or env variable.

**Describe alternatives you've considered**

Changing logs in the log collector - stupid thing to do as it is very complex and the logging libs provide it out of the box

**Additional context**
Probably it is just another configuration in the logger:
```
func formatLogger() {
	log.SetFormatter(&log.TextFormatter{
		DisableColors: true,
		FullTimestamp: true,
	})
	logger = log.WithFields(log.Fields{
		"component":   "akv2k8s",
		"application": "env-injector",
	})
}
```

Based on https://github.com/sirupsen/logrus, changing it with a config flag would result in:

```
func formatLogger() {
	log.SetFormatter(&log.JSONFormatter{
		DisableColors: true,
		FullTimestamp: true,
	})
	logger = log.WithFields(log.Fields{
		"component":   "akv2k8s",
		"application": "env-injector",
	})
}
```

**Comment by torresdal** [comment]
I agree @MarianZoll . A pull request by someone would really help speed this up... 😉 Adding to backlog for now.

**Comment by MarianZoll** [comment]
@torresdal We are working on it :) Unfortunately, we need to get Mgmt approval from IBM prior to contributing to OSS projects which are not part of CNCF or similar. But we'll try to push for it. 

In the meantime thx for taking the FR into consideration!

**Comment by torresdal** [comment]
This is now implemented in v1.1.0 just released
